l5-operator/config/samples/pets_v1_bestie.yaml

```diff
@@ -3,6 +3,6 @@ kind: Bestie
 metadata:
   name: bestie
 spec:
-  size: 2
+  size: 5
```

Comment:
Should we start with 5 replicas? Or just 1? How much CPU / Memory should we have in order to get the application running?